Directory.Packages.props

```diff
@@ -11,7 +11,7 @@
     <PackageVersion Include="CommunityToolkit.WinUI.UI.Controls" Version="7.1.2" />
     <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.5.240227000" />
     <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.26031-preview" />
-    <PackageVersion Include="Octokit" Version="10.0.0" />
+    <PackageVersion Include="Octokit" Version="13.0.1" />
```

Comment:
@NoobNotFound what is Octokit.NET even used for?

Reply:
basically nothing. (actually for the updater which download updates through github releases)